tutor/fmt.py

```diff
@@ -46,6 +46,14 @@ def alert(text: str) -> str:
     return click.style("⚠️  " + text, fg="yellow", bold=True)
 
 
+def warning(text: str) -> str:
```

Comment:
What's the purpose of `warning` when we already have `alert`?

Reply:
Alert prints to standard error and this seemed like a situation where we just wanted to display a warning rather than an error.

Reply:
Let's keep an alert. We don't want to write to stdout because it might mangle output from other commands, such as `tutor config printvalue/printroot`. And if we write to stderr than we are better off just using alert.